homeassistant/components/ozw/services.py

```diff
@@ -37,6 +43,99 @@ def async_register(self):
             ),
         )
 
+        self._hass.services.async_register(
+            const.DOMAIN,
+            const.SERVICE_SET_CONFIG_PARAMETER,
+            self.async_set_config_parameter,
+            schema=vol.Schema(
+                {
+                    vol.Optional(const.ATTR_INSTANCE_ID, default=1): vol.Coerce(int),
+                    vol.Required(const.ATTR_NODE_ID): vol.Coerce(int),
+                    vol.Required(const.ATTR_CONFIG_PARAMETER): vol.Coerce(int),
+                    vol.Required(const.ATTR_CONFIG_VALUE): vol.Any(
+                        vol.Coerce(int), cv.string
+                    ),
+                }
+            ),
+        )
+
+    @callback
+    def async_set_config_parameter(self, service):
+        """Set a config parameter to a node."""
+        instance_id = service.data[const.ATTR_INSTANCE_ID]
+        node_id = service.data[const.ATTR_NODE_ID]
+        param = service.data[const.ATTR_CONFIG_PARAMETER]
+        selection = service.data[const.ATTR_CONFIG_VALUE]
+
+        node = self._manager.get_instance(instance_id).get_node(node_id).values()
+
+        for value in node:
+            if (
+                value.command_class == CommandClass.CONFIGURATION
+                and value.index == param
+            ):
+                if value.type == ValueType.BOOL:
+                    value.send_value(str(selection == "True"))
+                    _LOGGER.info(
+                        "Setting configuration parameter %s on Node %s with bool selection %s",
+                        param,
+                        node_id,
+                        str(selection),
+                    )
+                    return
+                if value.type == ValueType.LIST:
+                    # accept either string from the list value OR the int value
+                    if isinstance(selection, int):
```

Comment:
```python
if not isinstance(selection, int):
    continue
```

Reply:
This one's for determining between int and string, I'll trim the indention down.